x-pack/plugins/security_solution/public/common/components/endpoint/host_isolation/endpoint_host_isolation_status.tsx

```diff
@@ -6,17 +6,23 @@
  */
 
 import React, { memo, useMemo, useRef, useEffect } from 'react';
-import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiTextColor, EuiToolTip } from '@elastic/eui';
+import { EuiBadge, EuiTextColor } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { useTestIdGenerator } from '../../../../management/hooks/use_test_id_generator';
 import { useIsExperimentalFeatureEnabled } from '../../../hooks/use_experimental_features';
+import { AgentPendingActionStatusBadge } from '../agent_pending_action_status_badge';
 
 export interface EndpointHostIsolationStatusProps {
   isIsolated: boolean;
-  /** the count of pending isolate actions */
-  pendingIsolate?: number;
-  /** the count of pending unisolate actions */
-  pendingUnIsolate?: number;
+  pendingActions: {
+    /** the count of pending isolate actions */
+    pendingIsolate?: number;
```

Comment:
Suggestion: As the parent key contains the word `pending`, I would remove it from the child:
```Typescript
{
 pendingActions: {
  isolate?: number;
  unisolate?: number;
   ...
 }
}
```

Reply:
@dasansol92  Yeah, I actually opened a separate issue to track refactoring this component so that the `pendingActions` are NOT hardcoded (see OLM issue 4507).  Ideally, this object would support anything the API returns for pendingActions and we should just loop through it to build the output.